recipes/dev-lxc.rb

```diff
@@ -2,10 +2,10 @@
 
 package 'lxc-dev'
 
-execute "install dev-lxc gem" do
-  command "chef gem install dev-lxc"
-  user "root"
-  environment( { "HOME" => "/root" } )
+execute 'install dev-lxc gem' do
+  command 'chef gem install dev-lxc'
+  user 'root'
+  environment('HOME' => '/root')
```

Comment:
I'm not sure that it was the right thing to remove the braces here so I'll add them back in a separate commit.